<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>591</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>ajouter</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>20</y>
        <width>226</width>
        <height>206</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>id client</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>nom client</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>prenom client</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_3"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>telephone</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_4"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>mail</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_5"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>date naissance</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_7"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>date inscription</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_8"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>100</y>
        <width>95</width>
        <height>64</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="ajoutcl">
         <property name="text">
          <string>ajouter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="modifcl">
         <property name="text">
          <string>modifier</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sexe</string>
      </property>
     </widget>
     <widget class="QComboBox" name="sexe">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>230</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Homme</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Femme</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>afficher</string>
     </attribute>
     <widget class="QTableView" name="tabcl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>581</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>380</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher Par Nom</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>380</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier Par Nom</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>420</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>GÃ©nerer un ficher Exel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_stat">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>380</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Statistique</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>supprimer</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>291</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>id client</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_6"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="supcl">
         <property name="text">
          <string>supprimer</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QLabel" name="ard">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>140</y>
        <width>251</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>OFF Capteur 1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>200</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>OFF Capteur 2</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>827</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
